<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Моделирование очереди</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="teLog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>1</y>
      <width>177</width>
      <height>510</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTicketsNum">
       <property name="text">
        <string>Число заявок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTicketsNum">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblGenExpVal">
       <property name="text">
        <string>Матожидание генератора</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leGenExpVal">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblProcExpVal">
       <property name="text">
        <string>Матожидание обработчика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leProcExpVal">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblProcDiap">
       <property name="text">
        <string>Полуинтервал обработчика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leProcDiap">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblQueueSize">
       <property name="text">
        <string>Размер очереди</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leQueueSize">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblDt">
       <property name="text">
        <string>delta_t</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDeltaT">
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEps">
       <property name="text">
        <string>Eps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEps">
       <property name="text">
        <string>0.005</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbWithoutReturn">
       <property name="text">
        <string>Без возврата</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbWithReturn">
       <property name="text">
        <string>С возвратом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDt">
       <property name="text">
        <string>Протяжка deltaT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbEvent">
       <property name="text">
        <string>Событийная протяжка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
